appveyor.yml

```diff
@@ -15,6 +16,13 @@ skip_tags: true
 
 clone_folder: c:\projects\habitat
 
+environment:
+  matrix:
+    - hab_build_action: build
+      hab_components: hab
+    - hab_build_action: test
+      hab_components: "core;http-client;builder-protocol;builder-depot-client;common"
+
 install:
   - ps: ./components/hab/win/win-build.ps1
```

Comment:
After the build?  It'll only exist after the cargo build, not in the test cycle.